server/graphql/v1/types.js

```diff
@@ -34,6 +34,7 @@ import { isUserTaxFormRequiredBeforePayment } from '../../lib/tax-forms';
 import { getCollectiveAvatarUrl } from '../../lib/collectivelib';
 import * as commonComment from '../common/comment';
 import { canViewExpensePrivateInfo, getExpenseAttachments } from '../common/expenses';
+import { PayoutMethodType as PayoutMethodTypes } from '../../models/PayoutMethod';
```

Comment:
Should we rename the Enum?

Reply:
Yes, I just saw that we don't have a perfect consistency on that but mist of the time it's plural, so let's have plural.